core/testsuite/src/main/java/com/blazebit/persistence/testsuite/AbstractCoreTest.java

```diff
@@ -368,6 +348,10 @@ protected String singleValuedAssociationIdNullnessPath(String path, String id) {
             return path;
         }
     }
+
+    protected String resolveRegisteredFunctionName(String functionName) {
+        return getIgnoreCase(getRegisteredFunctions(), functionName);
```

Comment:
I could do this:
```java
getIgnoreCase(STATIC_EMF_INTEGRATOR.getRegisteredFunctions(emf).keySet(), functionName)
```
or alternatively, I need to create a separate case insensitive map from `STATIC_EMF_INTEGRATOR.getRegisteredFunctions(emf)`.